config.py

```diff
@@ -6,7 +6,8 @@
 
 # Define and set variables that are required in the config file.
 core_config_keys = [
-    "FLPRODUCT", "FLPRODUCTi", "TARGETVERSION", "LOGOS_RELEASE_VERSION",
+    "FLPRODUCT", "FLPRODUCTi", "TARGETVERSION", "TARGET_RELEASE_VERSION",
+    "current_logos_version", "FLPRODUCT", "TARGETVERSION",
     "INSTALLDIR", "WINETRICKSBIN", "WINEBIN_CODE", "WINE_EXE",
```

Comment:
Looks like there's some redundancy here (`TARGETVERSION` and `FLPRODUCT`).

Also, since the purpose of the config file is to easily re-install with the same parameters, is it necessary to store `current_logos_version` there? I also think `FLPRODUCTi` could go, since it's never read from the config file as far as I can tell (`$ grep -n FLPRODUCTi *.py`) and is automatically derived from `FLPRODUCT` in `installer`.